README.md

```diff
@@ -100,6 +100,12 @@ You can now install the latest beta version of minishift.
   $ brew cask install minishift-beta
 ```
 
+### Un-installing Minishift
+
+.1 Try to delete the Minishift VM: `minishift delete`
```

Comment:
This is not rendering as expected. See below
![screenshot from 2017-02-15 00-33-48](https://cloud.githubusercontent.com/assets/1132451/22944373/7e9d309e-f316-11e6-9fb9-30740e55e668.png)